README.md

```diff
@@ -1,48 +1,132 @@
+# Node.js Help
+
 <p align="center">
   <a href="https://nodejs.org/">
-    <picture>
-      <source media="(prefers-color-scheme: dark)" srcset="https://nodejs.org/static/logos/nodejsStackedLight.svg">
-      <img src="https://nodejs.org/static/logos/nodejsStackedDark.svg" width="200px">
-    </picture>
```

Comment:
removing the picture here will cause light mode / dark mode problems. the change seems tangental to the PR title